app/forms/support_interface/application_forms/pick_course_form.rb

```diff
@@ -0,0 +1,48 @@
+module SupportInterface
+  module ApplicationForms
+    class PickCourseForm
+      include ActiveModel::Model
+      include ValidationUtils
+
+      attr_accessor :course_code, :application_form_id, :candidate_id, :course_option_id
+
+      validates :course_option_id, presence: true
```

Comment:
Suggest an error message of something like "Select a course to add to the application" rather than "can't be blank"?